about.html

```diff
@@ -0,0 +1,105 @@
+        <a class="about-me" href="#">About</a>
+        <a href="#">Program</a>
+        <a href="#">Join</a>
+        <a href="#">Sponsor</a>
+        <a href="#">News</a>
+        <a href="#">Campaign</a>
+      </div>
+    </div>
+    
+  </header>
+  <main>
+    <section class="A-first">
+      <div class="A-intro">
+        <h3>"Hello! It is Here Again"</h3>
+        <h2>NAIROBI INNOVATION WEEK EDITION 3 2022</h2>
+        <p class="short-desc">The inter-continental conference brings togethernthe community of experts, academics and activists who comprise of the Creative Commons affiliate network in a different country every two. This year, we hope to expand our invitation list including organizations and individuals. <span class="long-desc"> who want ot work with us in shared projects that advance the cause of commons free culture and open knowledge. So if you are active and engaged in the worlds of open content and knowledge-free software advocates  Wikipedians, Open knowledge, gallaries, libries, museums, archives, governments and fundations, lawyers and activists- we hope you'll consider joining us this year to build stronger, more vibtrant commons together.</span></p>
+        <p class="event-email">Please contact us per the Email for any further Questions about Nairobi Innovation Week Conference 2022!</p>
+        <p class="email-line"><a href="#">uonbinnovation.ac.ke</a></p>
+      </div>
+    </section>
+    <section class="logo-make">
+        <h2>Nairobi Innovation Week 2022 Logo</h2>
+        <hr class="hr-logo">
+        <p class="para1">The logo for the 2022 Nairobi Innovation week was decided through the logo competition from 1 April to 10.April by the students of the University.After Online pulic voting ad Selection Committee process, the logo from Peter Nguka won the prize.After consultation with Mr Nguka we optimized the prize winning-logo for this year's conference.</p>
+
+        <p class="para2">The logo of the conference 2022 was decided through the logo competition.</p>
+
+        <div class="logo-img">
+          <img src="./imgs/niw2019.jfif" alt="Event Logo">
+        </div>
+    </section>
+
+    <section class="A-third">
+      <h3>See the past Nairobi Innovation Week</h3>
+      <hr>
+      <p>Take a look at the last two events which took place in Kisumu and Mombasa respectively.</p>
+      <div class="last-events">
+        <a href="https://www.youtube.com/watch?v=-M4SNHmqWG8"><img src="./imgs/photo18.jfif" alt=""></a>
+        <a href="https://www.youtube.com/watch?v=-yToP3YiR0I"><img src="./imgs/photo20.jfif" alt=""></a>
+      </div>
+    </section>
+
+    <section class="A-four">
+      <h2>Partner</h2>
+      <hr>
+      <div class="A-sponsors">
+        <img src="./imgs/kcb.png" alt="KCB bank">
+        <img src="./imgs/safaricom.png" alt="Safaricom">
+        <img src="./imgs/microsoft.png" alt="microsoft">
+        <img src="./imgs/udemy.png" alt="Udemy">
+      </div>
+    </section>
```

Comment:
- [ OPTIONAL] Great job so far 👏🏻 , however, you could improve this section by using semantic  HTML code what do you think to use a list tag  `ul > li`.  it's always a good practice to use `li` to represent an item in a list and a `ul ` to group them,  It could be something like this :point_down: 
    ```HTML 
          <ul>
                <li><a class="about-me" href="#">About</a></li>
                <li><a href="#">Program</a></li>
                <li> ....</li>
          </ul>
     ```

- I know you can do it 💪🏻 😄